tidb-cloud/v7.1.0-performance-benchmarking-with-tpcc.md

```diff
@@ -0,0 +1,86 @@
+---
+title: TiDB Cloud TPC-C Performance Test Report
+---
+
+# TiDB Cloud TPC-C Performance Test Report
+
+## Test overview
+
+This test aims at showing the TPC-C performance of TiDB v7.1.0 in the Online Transactional Processing (OLTP) scenario.
+
+## Test environment (TiDB Cloud AWS)
+
+### Hardware configuration
+
+| Service type | EC2 type | Instance count |
+|:----------|:----------|:----------|
+| PD        | 4C8G-20G |     3     |
+| TiKV      |16C64G-1000G|     3     |
+| TiDB      | 16C32G|     2     |
+| TPC-C  | 8c16g|     1     |
+
+### Software version
+
+| Service type | Software version  |
+| :----------- | :---------------- |
+| PD           | v7.1.0  |
+| TiDB         | v7.1.0 |
+| TiKV         | v7.1.0 |
+
+### Parameter configuration
+
+#### TiKV parameter configuration
+
+```yaml
+[raft-engine]
+prefill-for-recycle = true
+```
+
+### Prepare test data and run tpcc
+
+1. Deploy TiDB v7.1.0 using TiDB Cloud.
+2. Connect the cluster and create a database named `tpcc`: 
+   ![privatelink](/media/tidb-cloud/privatelink.png)
```

Comment:
Is it necessary for users to use a [private endpoint]((https://docs.pingcap.com/tidbcloud/set-up-private-endpoint-connections)) when connecting to TiDB Cloud and executing the `go-tpc tpcc` command? Can they use the standard connection or VPC peering instead?

Reply:
For performance testing, the private endpoint is recommended.